agent/utils/version/postgresql.go

```diff
@@ -30,15 +34,45 @@ func ParsePostgreSQLVersion(v string) string {
 		return ""
 	}
 
-	parts := strings.Split(m[1], ".")
-	switch len(parts) {
-	case 1: // major only
-		return parts[0]
-	case 2: // major and patch
-		return parts[0]
-	case 3: // major, minor, and patch
-		return parts[0] + "." + parts[1]
-	default:
-		return ""
+	return m[1]
```

Comment:
this change breaks test on line https://github.com/percona/pmm/pull/1521/files#diff-7cc90b7313706bd058a0be08fa6588b152d8ddde9bf9f4229c16dddf59ebbdd9R30

Reply:
it might break logic on pmm-agent side

Reply:
move back to previous version